src/components/BaseNavigation.vue

```diff
@@ -21,6 +21,11 @@ export default {
 			required: true,
 		},
 	},
+	computed: {
+		isBorderNeeded() {
+			return (index) => (index < this.tabs.length - 1 ? "border" : "")
```

Comment:
`return (index) => ({ border:  index < this.tabs.length - 1 })` looks better isn’t it?